src/gui/dialogs/lua_interpreter.cpp

```diff
@@ -321,7 +321,7 @@ class tlua_interpreter::input_model {
 #ifdef HAVE_HISTORY
 		// Do history expansions
 		char * cmd_cstr = new char [cmd.length()+1];
-		std::strcpy (cmd_cstr, cmd.c_str());
```

Comment:
I'm seeing that error on Travis as well, so I might actually push this particular fix regardless of the status of this PR.